models/hashflow/optimism/hashflow_optimism_crosschain_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{
+    config(
+        alias = 'cross_chain_trades'
+        ,schema = 'hashflow_optimism'
+        ,partition_by = ['block_date']
+        ,materialized='incremental'
+        ,incremental_strategy = 'merge'
+        ,unique_key = ['block_date', 'source_chain', 'tx_hash']
+        ,post_hook='{{ expose_spells(\'["optimism"]\',
```

Comment:
It seems that `hashflow_optimism.cross_chain_trades` is relatively independent, as it is not included in `dex.trades` nor `dex_aggregator.trades`.